lib/money/money.rb

```diff
@@ -1,77 +1,107 @@
-require 'bigdecimal'
-require 'bigdecimal/util'
-
 class Money
   include Comparable
 
-  attr_reader :value, :cents
+  attr_reader :value, :cents, :currency
 
   class << self
-    def new(value = 0, _currency = nil)
+    attr_accessor :parser, :default_currency
```

Comment:
I think the `default_currency` class attribute could be dangerous, if this gem is used by another web framework that supports threading, one thread could set the default currency and impact another separate thread. Do we need `default_curency` being settable?